explorer/graphql/app/schema.graphql

```diff
@@ -230,24 +226,40 @@ type PageInfo {
   HasPrevPage: Boolean!
 }
 
+"""
+The query type represents all of the entry points into our object graph.
+"""
 type Query {
+  "Retrieve a specific XBlock by source chain ID and height."
   xblock(sourceChainID: BigInt!, height: BigInt!): XBlock
+
+  "Retrieve a range of XBlocks by source chain ID and height."
   xblockrange(from: BigInt!, to: BigInt!): [XBlock]!
+
+  "Get the total count of XBlocks."
   xblockcount: BigInt
+
+  "Retrieve a range of XMsgs by source chain ID and height."
   xmsgrange(from: BigInt!, to: BigInt!): [XMsg]!
```

Comment:
Do we use these? 

`xblockrange` in particular seems a little useless. Like - `from` and `to` what? we have many source chains. We'd need to at least add source chain (also comment isn't representative)

If we don't use any queries I say we remove

Reply:
XBlock has height so it's for example from height 5 to height 15. Regardless of source chain id. Does that make sense?

Good catch on the comment - I'll fix it.

Reply:
`xmsgrange` seems to be used by the UI.

Reply:
I belive this is used here https://github.com/omni-network/omni/blob/7e228e9a04cd9cc0aae326378d0444ca6c042f65/explorer/ui/app/routes/_index.tsx#L51

I think `to` and `from` are ID fields, so, you get the max count of xmsgs first, then `to` becomes that and `from` is `to` minus the number of messages you want to have